frontend/www/js/omegaup/components/arena/ClarificationList.vue

```diff
@@ -0,0 +1,248 @@
+              </button>
+            </div>
+            <div class="modal-body">
+              <label>
+                {{ T.wordsProblem }}
+                <select v-model="newClarification.problemAlias">
+                  <option
+                    v-for="problem in contestProblems"
+                    v-bind:value="problem.alias"
+                  >
+                    {{ problem.title }}</option
+                  >
+                </select>
+              </label>
+              <textarea
+                class="w-100"
+                maxlength="200"
+                v-model="newClarification.message"
+                v-bind:placeholder="T.arenaClarificationCreateMaxLength"
+              ></textarea>
+            </div>
+            <div class="modal-footer">
+              <button
+                type="button"
+                class="btn btn-secondary"
+                data-dismiss="modal"
+              >
+                {{ T.wordsClose }}
+              </button>
+              <button
+                type="button"
+                class="btn btn-primary"
+                v-on:click.prevent="sendClarification"
+              >
+                {{ T.wordsSend }}
+              </button>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+    <div class="table-responsive">
+      <table class="table mb-0">
+        <thead>
+          <tr>
+            <th class="text-center" scope="col" v-if="inContest">
+              {{ T.wordsContest }}
+            </th>
+            <th class="text-center" scope="col" v-else="">
+              {{ T.wordsProblem }}
+            </th>
+            <th class="text-center" scope="col">{{ T.wordsAuthor }}</th>
+            <th class="text-center" scope="col">{{ T.wordsTime }}</th>
+            <th class="text-center" scope="col">{{ T.wordsMessage }}</th>
+            <th class="text-center" scope="col">{{ T.wordsResult }}</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr v-for="(clarification, idx) in clarifications">
+            <td class="text-center align-middle">
+              {{
+                inContest
+                  ? clarification.contest_alias
+                  : clarification.problem_alias
+              }}
+            </td>
+            <td class="text-center align-middle">{{ clarification.author }}</td>
+            <td class="text-center align-middle">
+              {{ time.formatDateTime(clarification.time) }}
+            </td>
+            <td class="align-middle">
+              <pre>{{ clarification.message }}</pre>
+            </td>
+            <td class="align-middle">
+              <pre v-if="clarification.answer">{{ clarification.answer }}</pre>
+              <form class="form-inline justify-content-between">
+                <div class="form-group">
+                  <select
+                    class="form-control"
+                    v-model="clarificationsResponses[idx].selectedOption"
```

Comment:
```suggestion
                    v-model="clarification.selectedOption"
```

esto funciona? igual todos los demás lugares donde se usan `clarificationsResponses[idx]` o `idx`.

Reply:
No funciona.
Qué tal si hago un `localClarifications` y trabajo solo con ese?

Reply:
Aunque me parece que las computed properties no pueden modificarse :/

Reply:
Deja reviso bien.